### DISCUSSION TITLE
How to check which database is Prisma Client connected to?

### DISCUSSION THREAD

**Discussion by moroshko** [prisma/prisma#7467] [Q&A]
I have a script that initializes a Prisma Client and does some database operations.

```
const { PrismaClient } = require("@prisma/client");
const prisma = new PrismaClient();

// do something
```

I'm using `dotenv-flow` like suggested [here](https://github.com/prisma/prisma/issues/3865#issuecomment-825517105) to differentiate between `test` and `prod` databases.

I'd like to print out which database the Prisma Client is connected to after its initialization to make sure that I'm interacting with the correct database.

What's the easiest way to achieve this?

**Comment by moroshko** [comment]
I guess I could just print out `process.env.DATABASE_URL` 😄

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
